stats.go

```diff
@@ -112,3 +114,25 @@ func (s *Stats) Snapshot() *Stats {
 	})
 	return snap
 }
+
+func (s *Stats) String() string {
```

Comment:
Yes, but I figured the best way to represent this was json due it being a map.  I could of just let `fmt.Sprintf("%v", stat)` do the work but it would of been less readable.